monitoring/zabbix_host.py

```diff
@@ -356,6 +363,28 @@ def link_or_clear_template(self, host_id, template_id_list):
         except Exception, e:
             self._module.fail_json(msg="Failed to link template to host: %s" % e)
 
+    # Update the host inventory_mode
+    def update_inventory_mode(self, host_id, inventory_mode):
+
+	if not inventory_mode:
+            return
+
+        if inventory_mode == "automatic":
+            inventory_mode = int(1)
+        elif inventory_mode == "manual":
+            inventory_mode = int(0)
+        else: # inventory_mode == "disabled":
```

Comment:
when I read the code the default seems to be `disabled` but the doc says 'none'?